tests/test_networks.py

```diff
@@ -417,6 +418,15 @@ class Model(BaseModel):
     assert str(Model(a=dsn).a) == dsn
 
 
+def test_sqlite_dsns() -> None:
+    class Model(BaseModel):
+        a: SQLiteDsn
+
+    example_db_path: str = 'tests/mocks/test_db.db'
+
+    assert str(Model(a=example_db_path).a) == example_db_path
+
+
```

Comment:
Happy to change how we're testing this if we don't want to include a dummy db file in a new `mocks` folder.